docker/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG OPENJDK_VERSION=8
 FROM eclipse-temurin:${OPENJDK_VERSION}-jre
 
 ARG BUILD_DATE
-ARG SPARK_VERSION=3.3.2
+ARG SPARK_VERSION=3.4.1
```

Comment:
@Fokko : Could you remove this and update the title?

Reply:
```suggestion
ARG SPARK_VERSION=3.3.2
```